cmd-dump.c

```diff
@@ -916,6 +921,18 @@ static void print_chrome_perf_event(struct uftrace_dump_ops *ops,
 				    struct uftrace_perf_reader *perf,
 				    struct uftrace_record *frs)
 {
+	uint64_t evt_id = frs->addr;
+
+	switch (evt_id) {
+	case EVENT_ID_PERF_COMM:
+		pr_out(",\n{\"ts\":0,\"ph\":\"M\",\"pid\":%d,"
+		       "\"name\":\"process_name\","
+		       "\"args\":{\"name\":\"%s\"}}",
+		       perf->tid, perf->u.comm.comm);
```

Comment:
Is it always a process name?

Reply:
Hi, I added a basic process name to this commit, but updated this by adding "tid" field in the later commit 1da3695. Please take a look at it again. Thanks.

Reply:
OK, it seems github displays the commits in an wrong order..